utils/formatDate.ts

```diff
@@ -66,7 +66,7 @@ export const getComplementedDate = (dateString: string): string => {
     targetYear -= 1
   }
 
-  return `${targetYear}/${month}/${date}`
+  return new Date(`${targetYear}-${month}-${date}`)
```

Comment:
戻り値の型が`Date`になったことにより文字列のフォーマットにこだわる必要がなくなったので、Dateクラスのコンストラクタに数値として渡してもいいかと思います（月が0始まりなので`-1`しないといけないのが微妙ですが）。

```suggestion
  return new Date(targetYear, month - 1, date)
```

Reply:
そうですね。私もmonthの `-1` になんとも気持ち悪さを感じるんですが、まあ、こういうものだと思えば、ってとこですね 😅 
それでも、ご提案いただいたソースのほうがすっきりしているのでいいですね！ありがとうございます！